src/analyses/call_graph.cpp

```diff
@@ -192,11 +192,17 @@ class function_indicest
 /// was set when this call-graph was constructed the edge will be annotated
 /// with the call-site set.
 /// \return grapht representation of this call_grapht
-call_grapht::directed_grapht call_grapht::get_directed_graph() const
+call_grapht::directed_grapht
+call_grapht::get_directed_graph(const goto_modelt &goto_model) const
 {
   call_grapht::directed_grapht ret;
   function_indicest function_indices(ret);
 
+  // To make sure we include unreachable functions we first create indices
+  // for all functions in the function map
+  for(const auto &f : goto_model.goto_functions.function_map)
+    function_indices[f.first];
```

Comment:
This function is only computing a different representation of the same underlying graph. Do your changes just fix up a bit of information that is present in  the original call graph, or will it actually yield a different graph?